vault/logical_system_paths.go

```diff
@@ -60,7 +59,7 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 		},
 
 		{
-			Pattern: "config/ui/headers/" + framework.GenericNameRegex("header"),
+			Pattern: "config/ui/headers/" + framework.GenericNameRegex("header") + framework.OptionalParamRegex("multivalue"),
```

Comment:
Wouldn't it be more natural to make this an HTTP param (i.e. `?multivalue=true`) rather than making it part of the path?

Reply:
Sounds good. How do I do that? Couldn't find an example in the code.

Reply:
I believe you simply leave it out of the Pattern field, and define it as a field such as you have below.

Reply:
Found an example: https://www.vaultproject.io/api-docs/system/pprof#profile

Reply:
Thanks! That worked.